mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -93,7 +93,7 @@ def _find_all_labels(self):
         This method finds all existing labels in the repo
         :return A set of all labels which have been extracted from the repo
         """
-        url = 'https://api.github.com/repos/{}/{}'.format(self.repo, "labels")
+        url = f'https://api.github.com/repos/{self.repo}/labels'
```

Comment:
Cool! I didn't know this feature!

Reply:
This is available only in python 3.7. Does the environment where the label bot run support this version?

Reply:
Oh, sorry, it's indeed 3.6. Cool.